tests/test_utils/tests.py

```diff
@@ -213,6 +213,11 @@ def test_with_client(self):
             self.client.get("/test_utils/get_person/%s/" % person.pk)
             self.client.get("/test_utils/get_person/%s/" % person.pk)
 
+    def test_callstack_captured(self):
+        with self.assertNumQueries(1) as queries:
+            Person.objects.count()
+        self.assertIn('traceback', queries.connection.queries[0])
```

Comment:
I guess it would be nice to test the contents of 'traceback' rather than just that the context exists.